.github/workflows/garm-docker-image.yaml

```diff
@@ -0,0 +1,38 @@
+name: Build and push garm docker image
+on: workflow_dispatch
+
+jobs:
+  Build-and-Push-Image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    env:
+      REGISTRY: ghcr.io
+      IMAGE_NAME: confidential-containers/garm
```

Comment:
@wainersm can we push this to quay.io/confidential-containers/garm (and hence get a push secret in this repo)?

Reply:
@mkulke I think that push to ghcr.io is a better idea. I explain: there are people suggesting to move the CoCo images from quay.io to ghcr.io with the goal to easy the release of images (i.e. don't need to handle registry secrets), and we might end up doing that soon or later. Apart from that, the garm image won't be consumed by end users of CoCo so IMHO it is completely fine to keep it on ghcr.io